### ISSUE TITLE
Colour contrast fails AA 

### ISSUE THREAD

**Issue by Cheesycats** [NHSDigital/data-viz-community-of-practice#6]
Colour contrast for the below quote fails AA standards

![image](https://github.com/user-attachments/assets/11cfa838-d339-4e1b-9073-67d705b548a8)

![image](https://github.com/user-attachments/assets/d15736ae-7bea-4993-9d26-5f0f24bc5f2f)
